libs/aws/pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "langchain-aws"
-version = "0.2.13"
+version = "0.2.14"
```

This shouldn't be part of the PR, we do this before a release so it correctly reflects the release version.
```suggestion
version = "0.2.13"
```

---

will remove